tests/golden/defaults/cert-exporter/cert-exporter/10_helmchart/x509-certificate-exporter/templates/clusterrole.yaml

```diff
@@ -25,3 +25,15 @@ rules:
       - get
       - watch
       - list
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/instance: cert-exporter
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/name: x509-certificate-exporter
+    app.kubernetes.io/version: 2.12.0
+    helm.sh/chart: x509-certificate-exporter-1.20.0
+  name: cert-exporter-x509-certificate-exporter-node
+rules: null
```

Comment:
If the ClusterRole has no rules, is she required?

Reply:
This is a generated yaml from the tests.
But good question; this is generated from helm 🤷

Reply:
If it's generated, it makes sense to keep it as is and not manually changing it. Fine by me.